sound/soc/intel/boards/sof_sdw_rt_dmic.c

```diff
@@ -15,6 +15,7 @@
 static const char * const dmics[] = {
 	"rt715",
 	"rt712-sdca-dmic",
+	"rt722-sdca",
```

Comment:
can you add a comment on why we don't have rt715-sdca in the list? I am a bit lost on conventions.

Reply:
> can you add a comment on why we don't have rt715-sdca in the list? I am a bit lost on conventions.

Because rt715-sdca codec driver uses [rt715-aif1](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/codecs/rt715-sdca.c#L936), too. I will add a comment on it. BTW, if rt715-sdca codec driver can change its dai name to `rt715-sdca-aif1` and `rt715-sdca-aif2`, it would help machine driver to distinguish rt715 and rt715-sdca DAIs. What do you think @shumingfan?

Reply:
@bardliao I think the rt715-sdca codec driver could change the dai name to `rt715-sdca-aif1` and `rt715-sdca-aif2`.
The rt711-sdca codec driver did the same thing.